components/tyk-dashboard/templates/_helpers.tpl

```diff
@@ -0,0 +1,91 @@
+{{- /* vim: set filetype=mustache: */}}
+{{- /*
+Expand the name of the chart.
+*/}}
+{{- define "tyk-dashboard.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- /*
+ok
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "tyk-dashboard.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- /*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "tyk-dashboard.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "tyk-dashboard.gwproto" -}}
+{{- if .Values.gwTls -}}
+https
+{{- else -}}
+http
+{{- end -}}
+{{- end -}}
+
+{{- define "tyk-dashboard.dash_proto" -}}
+{{- if .Values.tls -}}
+https
+{{- else -}}
+http
+{{- end -}}
+{{- end -}}
+
+{{- define "tyk-dashboard.dash_url" -}}
+{{ include "tyk-dashboard.dash_proto" . }}://dashboard-svc-{{ include "tyk-dashboard.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}
+{{- end -}}
+
+{{- define "tyk-dashboard.gateway_url" -}}
+{{ include "tyk-dashboard.gwproto" . }}://gateway-svc-{{ include "tyk-dashboard.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.gateway.service.port }}
```

Comment:
I don't see `gateway.service.port` field defined in `values.yaml` file. Please update this helper declaration accordingly